src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUNDARY = 101;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(NUMBER_BOUNDARY));
+        return ball.toString();
```

Comment:
return Ball itself, not string representation. This is more flexible. Maybe some day you will need ball itself, not just string. If you need string you can call toString in a caller method (though it's not needed in System.out.println - it's called under the hood)

Reply:
+

Reply:
ok